model-optimizer/extensions/middle/fusings.py

```diff
@@ -96,7 +96,7 @@ def find_and_replace_pattern(self, graph: Graph):
                 for_graph_and_each_sub_graph_recursively(graph, fuse_linear_ops)
                 for_graph_and_each_sub_graph_recursively(graph, lambda G: G.clean_up())
 
```

Comment:
consider option to remove `run_not_recirsively=True`

Reply:
Discussed today and decided not to touch this code much due to some risky parts. For example, `fw` and `argv` can not be initialized for some sub-graphs.